src/indra_cogex/apps/data_display/__init__.py

```diff
@@ -238,33 +239,43 @@ def statement_display():
 
         # Get statements and assign belief from the meta data
         stmt_iter = []
+        source_counts = {}
         for rel in relations:
+            # Remove statements with only medscan evidence if not logged in
+            if (
+                remove_medscan
+                and "medscan" in rel.data["source_counts"]
+                and set(json.loads(rel.data["source_counts"]).keys()) == {"medscan"}
```

Comment:
Since you’re parsing this twice maybe save into a variable

Reply:
I thought it might make the check faster, but thinking twice that's not the case here

Reply:
Resolved in https://github.com/bgyori/indra_cogex/pull/100/commits/114e288061792fb6301300d0041926658636e691